tools/rulegen/go.go

```diff
@@ -20,14 +22,13 @@ load("@rules_proto_grpc//{{ .Lang.Dir }}:repositories.bzl", rules_proto_grpc_{{
 
 rules_proto_grpc_{{ .Lang.Name }}_repos()`)
 
-
-var goLibraryRuleTemplateString = `load("//{{ .Lang.Dir }}:{{ .Rule.Base}}_{{ .Rule.Kind }}_compile.bzl", "{{ .Rule.Base }}_{{ .Rule.Kind }}_compile")
+var goLibraryRuleTemplateString = `load("//{{ .Lang.Dir }}:{{ .Rule.Base}}_proto_compile.bzl", "{{ .Rule.Base }}_proto_compile")
```

Comment:
Both rules now used `proto_compile` rule so they can share this prefix. gRPC rule needs some more loads, see below.